### ISSUE TITLE
`deno doc --lint` doesn't recognise JSDoc description if preceded by `@deprecation`

### ISSUE THREAD

**Issue by iuioiua** [denoland/deno_doc#420]
To reproduce:
1. Run `deno doc --lint https://deno.land/std@0.208.0/encoding/hex.ts`.
2. Observe that the linter claims that the [`encode()` function](https://deno.land/std@0.208.0/encoding/hex.ts?source=#L62) is missing a JSDoc comment.

The function isn't missing a JSDoc comment. Rather, the JSDoc comment isn't being recognised because a `@deprecated` block comes before the comment. The workaround is to move the comment to be the first block.

**Comment by crowlKats** [comment]
Yes, this goes for most if not all jsdoc tags: they are greedy and as such the description/body needs to always come before any tag.
